daemon/graphdriver/btrfs/btrfs.go

```diff
@@ -6,6 +6,7 @@ package btrfs
 #include <stdlib.h>
 #include <dirent.h>
 #include <btrfs/ioctl.h>
+#include <btrfs/ctree.h>
```

Comment:
We probably want to run this at least through a compile test on all our deb/rpm target platforms to make sure this header is available too.  (Isn't there a different Jenkins job that can test that now, too?)